packages/dapp/src/components/GlobalNav.tsx

```diff
@@ -8,31 +8,43 @@ export interface NavBarProps {
   balance: string;
   votingBalance: string;
   network: string;
+  userAccount: string;
 }
 
 const GlobalNavComponent: React.SFC<NavBarProps> = props => {
   const shouldRenderErrorBar = props.network !== "4";
   return (
     <>
-      <NavBar balance={props.balance} votingBalance={props.votingBalance} />
+      <NavBar
+        balance={props.balance}
+        votingBalance={props.votingBalance}
+        userAccount={props.userAccount}
+        buyCvlUrl="https://civil.co/cvl/"
```

Comment:
Not passing the counts for user challenges/votes?

Reply:
This is part of what I'm adding to the new ticket there's a placeholder in there now. https://app.clubhouse.io/civil/story/2100/dapp-nav-and-drawer-updates